source/fleetprovisioning/FleetProvisioning.h

```diff
@@ -186,6 +191,14 @@ namespace Aws
                      * @return returns false if client is not able to find the file or if valid permissions are not set
                      */
                     bool LocateDeviceKey(const std::string filePath);
+					
+					/**
+					 * \brief Build template parameters
+					 *
+					 * @param A JSON object specified as an escaped string
```

Comment:
We should also update the documentation here once there's a bit of error handling to describe the error conditions that one might expect from this method